### ISSUE TITLE
Spacing Issues on Newest Version

### ISSUE THREAD

**Issue by kmackinley** [SerpicoProject/Serpico#553]
# Please fill out the Bug Form or Feature Request Below

-------------------
## Bug
Describe the issue and steps to reproduce

1. Upload template
2. Generate report to find random spacing issues.

### Example Use Case

I have not been able to pinpoint this issue. However, since the new version, my uploaded templates randomly delete spaces in random areas. I've checked formatting on the Word templates and there is no formatting on some of the areas where the spaces are removed. As a painful workaround (only sometimes, not all) copying the text into Sublime and copying it back into the documents resolves the issue; suggesting formatting where there is none visible in the document (even when "showing formatting"). Again, this is only happening with the newest version.

I can provide examples if needed, but unsure as to how they would help since no markup language is used where the deletion of spaces happens.

**Comment by MaxNad** [comment]
Hi,
Just to make sure I understand, are you running the latest release or straight from master ?
Thanks.

**Comment by kmackinley** [comment]
This is from the latest release.

**Comment by Clickb8** [comment]
Hi,
Any update regarding this issue ? I have the same problem: Spaces are randomly deleted when generating a report. I am using the latest release too (from deb package) and a custom Word template.
Thanks

**Comment by kmackinley** [comment]
I'd like to note on this issue that it seems to be resolved in the source version, whereas the current release has these weird spacing issues.

**Comment by Sn00z3r** [comment]
Anybody have any idea which code part is responsible here, because I would like to "patch" our production serpico to fix this issue and I'm not really keen on just overwriting all files :)

**Comment by BuffaloWill** [comment]
@Sn00z3r have you tested the source version? This issue has been fixed in master. Monkey patching your code will have unintended consequences. 

That being said, the code responsible is `helpers/xslt_generatio.rb` and specifically this commit; https://github.com/SerpicoProject/Serpico/commit/456c45b6ad7b69a240463e759a935d84cf79b4cc#diff-1700bffbf6519338fbe0c4702ca11c15.

**Comment by jakxx** [comment]
Finding this issue myself as well. Would it be worth pushing a new release for this fix?

**Comment by BuffaloWill** [comment]
I agree we should push a new release. It is well past due, let me look at the timeline for this.

**Comment by jakxx** [comment]
Any more thoughts on getting this one (and others) into a new release?
